src/opnsense/service/templates/OPNsense/Unbound/core/blocklists.conf

```diff
@@ -51,6 +51,12 @@ custom_{{loop.index}}={{uri}}
 {%      endfor %}
 {%    endif %}
 
+{%    if not helpers.empty('OPNsense.unboundplus.dnsbl.localzone')%}
+[dnsbl]
```

Comment:
Probably can have a better designation here: „dnsbl“ is too broad

Reply:
Makes sense.  How about '[response]' since this config governs how unbound responds to blocklist matches?